apm-sniffer/optional-plugins/optional-spring-plugins/optional-spring-cloud/gateway-2.0.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/spring/cloud/gateway/v20x/HttpClientRequestInterceptor.java

```diff
@@ -45,6 +45,11 @@ public void beforeMethod(final EnhancedInstance objInst,
                              final Object[] allArguments,
                              final Class<?>[] argumentsTypes,
                              final MethodInterceptResult result) throws Throwable {
+        
+        if (!ContextManager.isActive()) {
+            return;
+        }
```

Comment:
Please give a clear comment at here about what are the cases there is no active context. 

I know you mentioned it in the PR description, but you should put a formal one here and others too.